dbms/src/Flash/Pipeline/PipelineExecStatus.h

```diff
@@ -0,0 +1,72 @@
+// Copyright 2023 PingCAP, Ltd.
```

Comment:
Same as `PipelineExecutor.*`, move them to `Executor` directory as it's query-scope concept and the top-level entrance.

Reply:
Also, consider renaming it to `PipelineExecutorStatus` to reflect the connection with `PipelineExecutor`. I.e., they are all query scope and top-level entrance.

Reply:
ok, has moved to `Executor/` and renamed to `PipelineExecutorStatus`